docs/basic-usage.md

```diff
@@ -81,6 +81,48 @@ a simple URL string, you can use a closure or command instead.
 $schedule->url('https://my-status-cloud.com?site=foo.com')->everyFiveMinutes();
 ```
 
+### Scheduling Queue Jobs
+
+If you want to schedule a Queue Job, you can use the `queue()` method and specify the queue name, job name and data your job needs:
+
+```php
+$schedule->queue('queue-name', 'jobName', ['data' => 'array'])->hourly();
+```
+
+!!! note
+
+    To learn more about the [Queue package](https://github.com/codeigniter4/queue) you can visit a project page.
```

Comment:
Sorry, what project page do you mean here?

Reply:
I'm referring to the linked GitHub repository. Does that wording seem unclear?

Reply:
Yes, it seems you're referring to another page. Maybe something like this:

To learn more about queues, you can visit the [Queue package](https://github.com/codeigniter4/queue).

Reply:
Ok, I will change it. Thank you.